### ISSUE TITLE
New SC Explorer Dashboard (catch-all ticket)

### ISSUE THREAD

**Issue by KuraFire** [sourcecred/sourcecred#2227]
This ticket is the overall ticket for the new, @decentralion-approved Explorer dashboard. See the image for the UX wireframe. 

Individual tickets for implementation:

- [ ] Top Chart #
- [ ] Timeframe Control  #
- [ ] Stats Circles #
- [ ] Activity Table #
- [ ] Plugin Bar Chart #

![Wireframes - SourceCred Explorer - HOME@2x](https://user-images.githubusercontent.com/33087/92644109-aa567e00-f297-11ea-8515-392c21d0cbb3.png)

**Comment by KuraFire** [comment]
# Important UX For Transition From Old Explorer To New Explorer UI

Between the development and implementation of CredRank and the new Explorer UI, the idea of implementing CredRank for the old (existing) Explorer "until the new UI is done" is considered ill-advised because most of that work would be throwaway very soon thereafter. 

To optimize development cycles, @decentralion suggested building the new Explorer UI on CredRank directly, not supporting / developing CredRank for the existing (old) Explorer UI. This would then create a parallel explorer (new UI) while leaving the existing one in place (old UI), until the new explorer is done and weather-proofed sufficiently for us.

Faruk (me) agrees with this approach being most effective and valuable use of time and dev cycles, and can be handled well from a UX perspective. @blueridger raised the concern about the confusion of there being two explorers for end users during this transition phase, and checked in with me to resolve that.

DESIGN PROPOSAL: 
I recommend we add a very clear, prominent message in the UI to explain that the _new_ explorer is in **BETA** and available for previewing, but does NOT YET touch Grain distributions at all. It should be abundantly clear the new UI is only for PREVIEWING purposes during this time, and for implementing (and previewing) the new CredRank algorithm. 

This message UI should also explain that some Cred calculations will be different between old explorer and new Explorer UI.

**OPTIONAL**: We can also include a modal dialog on the first page load of the new Explorer to explain all this with more text, and let the user set a cookie to prevent that modal from reoccurring. This would mean first use of cookies on the Explorer, which opens up GDPR-related concerns, so we may want to skip this and use a simpler top bar message on the new UI that persists and cannot be closed/hidden.

cc @topocount (dev lead).

**Comment by decentralion** [comment]
> This would mean first use of cookies on the Explorer, which opens up GDPR-related concerns, so we may want to skip this and use a simpler top bar message on the new UI that persists and cannot be closed/hidden.

Yeah, let's punt on cookies until we need them for something more substantial.
